apps/learning-snippets/src/test/tree-widget/Telemetry.test.tsx

```diff
@@ -135,7 +136,7 @@ describe("Tree-widget", () => {
           cleanup();
```

Comment:
[The docs](https://testing-library.com/docs/react-testing-library/api/#cleanup) say you shouldn't need to call this

Reply:
The problem is the tests have flaky warning `was not wrapped in act` without this.
https://github.com/testing-library/react-testing-library/issues/1231
https://github.com/testing-library/react-testing-library/issues/1216
similar issues are reported to react-testing-library.